students/21th/login.js

```diff
@@ -0,0 +1,31 @@
+const inputId = document.getElementById('id');
+const inputPw = document.getElementById('password');
+const inputItems = document.getElementById('inputContainer')
+const logInButton = document.getElementById('logInButton');
+const mainATag = document.getElementById(moveMain);
+const loginEventTrue = () => {
```

Comment:
```suggestion
const mainATag = document.getElementById(moveMain);

const loginEventTrue = () => {
```
구분되는 요소들끼리는 한줄 띄워주는 것이 가독성 측면에서 좋겠습니다!

Reply:
수정했습니다!